### ISSUE TITLE
Crash with the last update

### ISSUE THREAD

**Issue by NovaXCIV** [team-abnormals/allurement#46]
It crashes the game when i repair the anvil with iron ingots.

**Comment by bageldotjpg** [comment]
Post a crash report and your latest.log on a paste site

**Comment by SiverDX** [comment]
i recommend making the anvil mixin non-required, it clashes with apotheosis
(in general there are a lot of mods which do this modification so a lot of potential for incompat)

``` log
[01Oct2023 11:37:04.652] [pool-4-thread-1/WARN] [mixin/]: @ModifyConstant conflict. Skipping allurement.mixins.json:AnvilMenuMixin->@ModifyConstant::createResult(I)I with priority 1000, already redirected by apotheosis.mixins.json:AnvilMenuMixin->@ModifyConstant::apoth_removeLevelCap(I)I with priority 1000
```

not sure if there are more crash reasons

**Comment by NovaXCIV** [comment]
https://pastebin.com/qwafcXZw

**Comment by ModernGamingWorld** [comment]
Till its fixed just go back to allurement-1.19.2-3.1.0

**Comment by bageldotjpg** [comment]
> i recommend making the anvil mixin non-required, it clashes with apotheosis (in general there are a lot of mods which do this modification so a lot of potential for incompat)
> 
> ```
> [01Oct2023 11:37:04.652] [pool-4-thread-1/WARN] [mixin/]: @ModifyConstant conflict. Skipping allurement.mixins.json:AnvilMenuMixin->@ModifyConstant::createResult(I)I with priority 1000, already redirected by apotheosis.mixins.json:AnvilMenuMixin->@ModifyConstant::apoth_removeLevelCap(I)I with priority 1000
> ```
> 
> not sure if there are more crash reasons

This is unrelated and does not seem to be causing a crash, it just skips Allurement's mixin and uses Apotheosis's.
Edit: it does seem to be causing a crash but regardless, it is unrelated
